lib/runners/csharp.js

```diff
@@ -89,41 +89,6 @@ module.exports.run = function run(opts, cb) {
       // compile
       compile(args, function(error, stdout, stderr) {
         if (error) return fail(error, stdout, stderr);
-
-        const exe = path.join(opts.dir, 'runner.exe');
-
-        // compile([
```

Comment:
This was a WIP. I had just left it in so that I would either remember to complete it or someone else would think to run with it. 

I actually gave up on it in frustration with dealing with C# assembly references but had hoped to get back to it soon. If we were to clean this up then we would probably just want to clean up the files it references as well, so maybe its better to just leave it in. I know people always say we can resurrect it from Git history but that rarely actually happens - more likely it will be forgotten.